Proof-read Hoodie Tutorial

raquelxmoss opened the pull request:
This PR fixes a few broken links and converts some stray `<code>` blocks into markdown code blocks (with language flags for nice syntax highlighting). 

It also corrects a grammar mistake, and clarifies a few terms that might be ambiguous for a beginner reading this tutorial.

raquelxmoss committed:
Change <code> blocks to markdown code blocks

raquelxmoss committed:
Fix broken links

raquelxmoss committed:
Change <code> block to markdown code block

raquelxmoss committed:
Correct grammar and clarify a few terms

'Incognito mode' is known as 'Private mode' in some browsers, so I added 'or private mode' so that readers can find what you're talking about if they're not already familiar with private/incognito browsing.

HipsterBrown commented on en/tutorials/index.md:
```diff
@@ -11,7 +11,7 @@ If you experience any problems at any step of this doc, please check our <a href
 ### Prerequisites
 
-You should have [installed Hoodie and its prerequisites](/en/start/) and read the [first part of this guide](/en/start/getting-started/getting-started-1.html). We'll be looking at the Hoodie app you created in that.
+You should have [installed Hoodie and its prerequisites](/en/start/) and read the [first part of this guide](/en/start/getting-started/getting-started-1.md). We'll be looking at the Hoodie app you created in that.
```
> `[first part of this guide](/en/start/getting-started/getting-started-1.md)`

This link was pointing to an `.html` file before because these Markdown files get converted to HTML for the Docs website. It will need to be reverted back to `.html` so the link still works.

HipsterBrown commented on en/tutorials/index.md:
```diff
@@ -38,11 +38,11 @@ Go ahead, add a couple of todos!
 #### 1.3 Offline support
 
-[We strongly believe that apps shouldn't break just because they're offline](/en/hoodieverse/hoodie-concepts.html#offline-first). So naturally, this todo app also works offline. Give it a try:
+[We strongly believe that apps shouldn't break just because they're offline](/en/hoodieverse/hoodie-concepts.md#offline-first). So naturally, this todo app also works offline. Give it a try:
```
Same fix needed as the "first part of this guide" link. https://github.com/hoodiehq/documentation/pull/206/files#r58724226

jennwrites commented:
I've reviewed the changes and this pending the rendering issue @HipsterBrown mentioned, this looks great. Thanks @raquelxmoss LGTM

HipsterBrown commented:
Great work @raquelxmoss. I can see why you figured the links were broken but since documentation will be converted to HTML, they need to stay pointing to the final HTML files instead of the source Markdown.   LGTM once those links are reverted back to `.html`.  

Thanks for working on this! :clap:

raquelxmoss committed:
Revert links to html

raquelxmoss commented:
@HipsterBrown I've changed those links back to `.html` :smile:

raquelxmoss committed:
Change `html` blocks to `markup`

[As per the documentation](https://github.com/raquelxmoss/documentation#syntax-highlighting), `html` blocks are not supported, while `markup` blocks are.

HipsterBrown commented:
LGTM :+1: Will merge asap
